react_app/src/threeVisualizer.js

```diff
@@ -0,0 +1,230 @@
+import * as three from 'three';
+import React from 'react';
+import { TrackballControls } from 'three/examples/jsm/controls/TrackballControls.js';
+
+const IMAGE_SIZE = 64;
+
+class ImageParams {
+  constructor(size) {
+    this.width = size;
+    this.height = size;
+  }
+}
+
+class Atlas {
+  constructor(image, numImages, numRows, numColumns) {
+    this.image = image;
+    this.width = image.width * numColumns;
+    this.height = image.height * numRows;
+    this.numImages = numImages;
+  }
+}
+
+class Point {
+  constructor(x, y, z) {
+    function randomInt() {
+      const val = Math.random() * 700;
```

Comment:
Where did "700" or "-400" come from? Shouldn't they be calculated values from some scale param?
At least take them out as constants.

Reply:
Factored them out to be comments - default construction isn't used in the final product, and it's mostly useful for debugging